### PR TITLE
dark mode

### PR DESCRIPTION
**Pull request by dominiquefl**
### TL;DR 
Introduce ability to switch theme to dark mode. 

### How
Followed this tutorial: https://medium.com/@sylwiavargas/dark-mode-in-ruby-on-rails-with-cookies-c892ac52ebaa
- write cookie methods in the application controller 
- one to set the dark mode cookie and redirect users to welcome index path, the other to delete the cookie and redirect to welcome index path 
- created routes for the two methods 
- enabled the application/html/erb head to check cookies, and load the proper stylesheet 
- created nav buttons to enable/disable dark mode 
- created the dark stylesheet 

**Additional learning resources**
- The Odin Project https://www.theodinproject.com/courses/ruby-on-rails
- on cookies, sessions, and flashes: https://www.theodinproject.com/courses/ruby-on-rails/lessons/sessions-cookies-and-authentication
- on `redirect_back` method: https://blog.bigbinary.com/2016/02/29/rails-5-improves-redirect_to_back-with-redirect-back.html

### See it live
https://screenshot.click/05-02-12_23-23-38.gif
light mode: 
image

dark mode: 
image

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,2 +1,13 @@
...
+  def moon
+    cookies[:moon] = {
+      value: 'dark mode on'
+    }
+    redirect_to welcome_index_path
+  end
+
+  def sun
+    cookies.delete(:moon)
+    redirect_to welcome_index_path
```
Comment (dominiquefl) [inline app/controllers/application_controller.rb:11]: why are we redirecting back to the welcome_index_path? lets keep the user on the same page.

Comment (dominiquefl) [reply app/controllers/application_controller.rb:11]: https://blog.bigbinary.com/2016/02/29/rails-5-improves-redirect_to_back-with-redirect-back.html
